configs/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM envoyproxy/envoy:latest
```

Comment:
Should we perhaps name this file Quickstart_Dockerfile or something?

Reply:
I added some comments to the Dockerfile, since Docker autobuilder only supports the name Dockerfile, and I'm not sure if other stuff is hardcoded to Dockerfile.